### ISSUE TITLE
doc only works on some docstrings

### ISSUE THREAD

**Issue by retrogradeorbit** [epiccastle/spire#60]
```
user> (doc spire.default/set-ssh!)
-------------------------
spire.default/set-ssh!
([host-string-or-config])
  sets the current default connection context to the specified ssh
  connection
  (set-ssh! host-config)

  given:

  `host-config`: a host-string or host config hashmap
nil
user> (doc spire.module.shell/shell)
nil
```

**Comment by retrogradeorbit** [comment]
```
user> (clojure.repl/print-doc (meta #'spire.module.shell/shell))
-------------------------
spire.module.shell/shell
([& args])...
```

works. but:

```
user> (doc spire.module.shell/shell)
nil
```

does not. :thinking:

**Comment by retrogradeorbit** [comment]
fixed in 0.1.0-alpha.11
